scripts/tag_west_repos.sh

```diff
@@ -109,6 +119,11 @@ push_tag() {
     project="$1"
     tagname="$2"
 
+    if [ -z "$project" ] || [ -z "$tagname" ]; then
+	    echo "empty project or tagname" 1>&2
+	    return
```

Comment:
shouldn't this be `exit 1`?

Reply:
Well, we already do a test pass in `check_tags()` so this is just an extra guard.